assets/js/search/setActiveTab.js

```diff
@@ -0,0 +1 @@
+window.location.hash = document.getElementById('activeTab').value
```

Comment:
I said in that thread but I'm a bit concerned about accessibility issues around this... 
Changing page content dynamically could be confusing

I've asked someone else for their opinion but we'll leave in for now and can redesign if necessary

Reply:
I did ask Steph about this following the thread and she mentioned it was something we can look at following feedback on search.